deploy/src/Microsoft.Azure.IIoT.Deployment/Configuration/HelmSettings.cs

```diff
@@ -13,7 +13,10 @@ class HelmSettings {
         public static string _defaultRepoUrl = "https://microsoft.github.io/charts/repo";
         public static string _defaultChartVersion = "0.3.2";
         public static string _defaultImageTag = "2.7.206";
```

Comment:
Should we update this to 2.8.0?

Reply:
Maybe once it is present in MCR ? At the moment `2.8.*` versions are only present in our development ACRs and for consuming from those I would need to check-in credentials for them.